client/resource_group/controller/config.go

```diff
@@ -40,12 +40,13 @@ const (
 	// If we want a factor of 0.5 per second, this should be:
 	//
 	//	0.5^(1 second / mainLoopUpdateInterval)
-	movingAvgFactor                = 0.5
-	notifyFraction                 = 0.1
-	consumptionsReportingThreshold = 100
-	extendedReportingPeriodFactor  = 4
-	defaultGroupLoopUpdateInterval = 1 * time.Second
-	defaultTargetPeriod            = 10 * time.Second
+	movingAvgFactor                 = 0.5
+	notifyFraction                  = 0.1
+	consumptionsReportingThreshold  = 100
+	extendedReportingPeriodFactor   = 4
+	defaultGroupCleanupInterval     = 1 * time.Minute
```

Comment:
I think it may be longer time, like  `10 * time.Minute`? I wonder if there are too many clients in serverless.